autogpt_platform/backend/backend/executor/manager.py

```diff
@@ -155,18 +155,19 @@ def update_execution(status: ExecutionStatus) -> ExecutionResult:
     # changes during execution. ⚠️ This means a set of credentials can only be used by
     # one (running) block at a time; simultaneous execution of blocks using same
     # credentials is not supported.
-    credentials = creds_lock = None
+    creds_lock = None
     if CREDENTIALS_FIELD_NAME in input_data:
         credentials_meta = CredentialsMetaInput(**input_data[CREDENTIALS_FIELD_NAME])
         credentials, creds_lock = creds_manager.acquire(user_id, credentials_meta.id)
```

Comment:
Isn’t it used here?

Reply:
it's another assignment, so it's overridden